src/decorators/exclude.decorator.ts

```diff
@@ -0,0 +1,23 @@
+import { defaultMetadataStorage } from '../storage';
+import { ExcludeOptions } from '../interfaces';
+
+/**
+ * Marks property as excluded from the process of transformation. By default it excludes the property for both
+ * constructorToPlain and plainToConstructor transformations, however you can specify on which of transformation types
+ * you want to skip this property.
+ */
+export function Exclude(options: ExcludeOptions = {}) {
```

Comment:
We could try to add TS types for decorators. In this case `: PropertyDecorator | ClassDecorator`.